src/features/elementfeature.jl

```diff
@@ -1,6 +1,26 @@
 using ..ChemistryFeaturization.Utils.ElementFeatureUtils
 using DataFrames
 
+
+abstract type EncoderDecoder end
+
+"""
+    DummyED(encode_f, decode_f, nbins, logspaced)
+
+EncoderDecoder type which uses a dummy variable (as defined in statistical literature), i.e., which employs
+one-hot encoding and a one-cold decoding scheme.
+"""
+struct DummyED <: EncoderDecoder
```

Comment:
I get what you mean by this name, but I feel like OneHotED might be more evocative for most users? #ntih

Reply:
Yeah I initially started off by naming it `OneHotED` too, but then felt it doesn't really make sense since one-hot is only the encoding scheme. Would `OneHotOneCold` or `OneHotCold` be a better alternative to `DummyED`?

Reply:
Personally, I think OneHot would be fine. I see your point that it only describes encoding of course, but I see that as a smaller problem than a user misunderstanding `Dummy` to mean that they shouldn't use it at all because it's some kind of test or placeholder, which I feel like is the first connotation that comes to mind for that word...

Reply:
I agree, `DummyED` needs to be changed. I think `OneHotOneCold` might be a more appropriate name over `OneHot` though, considering how the name is literally the encoding scheme+ decoding scheme.

Reply:
yeah that's fine by me too...do we want `ED` at the end or do we think that's not necessary?

Reply:
> yeah that's fine by me too...do we want ED at the end or do we think that's not necessary?

I don't think it's necessary. I feel like having it kinda implicitly sets a naming convention of sorts, which might prove to be redundant as we have more types of `ED`...